### ISSUE TITLE
Windows Install

### ISSUE THREAD

**Issue by whitead** [BradyAJohnston/MolecularNodes#1]
Excited to see this! Blender wouldn't let me enable the plugin saying it crashed on line 23 of `__init__.py` file (see source below). I was able to just install atomium by copy/paste the install script in repo though. This was on Blender 3.0.0

```py
bl_info = {
    "name" : "Molecular Nodes",
    "author" : "Brady Johnston", 
    "description" : "A plugin and nodes for working with molecular data in Blender.",
    "blender" : (3, 0, 0),
    "version" : (0, 3, 2),
    "location" : "Perth, Australia",
    "waring" : "",
    "doc_url": "", 
    "tracker_url": "https://github.com/BradyAJohnston/MolecularNodes/issues", 
    "category" : "Molecular" 
}

import bpy
import bpy.utils.previews

import subprocess
import os
import bpy
import bpy
import os
import bpy
import atomium # error here
```

**Comment by BradyAJohnston** [comment]
Ah thank you for the bug report already! Really shouldn't be trying to import atomium until after the setup step etc. I will have a go at fixing it (and test better for installation!). 

Have you tested out the panel at all?

**Comment by whitead** [comment]
Yup, converted an MD trajectory into PDB and loaded it up. I messed around for a little while trying to get the geom nodes to play the frames but couldn't figure it out after a little while. Figured I'd wait for you to post a screenshot or some docs on how to use the added geometry nodes.

I'm really impressed with the new panel and nodes! I've been trying to figure out how to add my own geom nodes, like the math from [my symmetry project](https://github.com/whitead/symd), so this is a very useful demo for me to look at.

**Comment by BradyAJohnston** [comment]
I think I have fixed the Atomium issue and a couple of other minor things, will be updating shortly. 

Give this node setup a go, should get you started with animating between frames!

For the properties node, select the `*_properties` collection, and for the animate node choose `*_frames` collection, and the material inside of the `colour` node.

![image](https://user-images.githubusercontent.com/36021261/165194192-c5da34cf-cdb3-400a-9f41-4453a2eea694.png)

To begin with, all of the atoms will be the point-clouds, which are only visible in Cycles (there is a node in styling to create EEVEE meshes, but it is much less performant in the viewport).

For GN modifier node tree, you need to create an output and name it something (i.e. `colour`) to pass the colours out:
![image](https://user-images.githubusercontent.com/36021261/165194401-7217bbcb-3695-414a-bf20-866db37d173a.png)

And then in the shader for the material use that attribute:
![image](https://user-images.githubusercontent.com/36021261/165194439-67f3b31c-0345-4276-b164-e9cce17ff7ad.png)

Will be working on proper documentation (and a video!) for the nodes themselves.

This was all built with the [Serpens Addon](https://joshuaknauber.notion.site/Serpens-Documentation-d44c98df6af64d7c9a7925020af11233), which has made the whole thing 100000x easier than it would have otherwise been!

**Comment by whitead** [comment]
Great, thanks for pointing out Serpens. Will give this a try and get back to you. Thank you!

**Comment by BradyAJohnston** [comment]
Have also updated the README with additional instructions, and bumped the release that should enable correctly!
